tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -165,11 +165,18 @@ export async function loadTspConfig(typeSpecDirectory: string): Promise<Exclude<
 // e.g. sdk/mongocluster/arm-mongocluster
 export async function getGeneratedPackageDirectory(typeSpecDirectory: string, sdkRepoRoot: string): Promise<string> {
     const tspConfig = await loadTspConfig(typeSpecDirectory);
-    const serviceDir = tspConfig.parameters?.['service-dir']?.default;
+    let serviceDir = tspConfig.parameters?.['service-dir']?.default;
```

Comment:
so typespec sdk can't be used parse tspconfig?
since customer may config it option like `service-dir: {come-other-global-para}`